content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -74,22 +74,20 @@ Clone the [DataTrails SCITT Examples](https://github.com/datatrails/datatrails-s
     ISSUER="sample.synsation.io"
 
     # signing key to sign the SCITT Statements
-    SIGNING_KEY="/tmp/my-signing-key.pem"
+    SIGNING_KEY="my-signing-key.pem"
```

Comment:
Is there a danger that this will create secrets in the git repo that may then be accidentally pushed to github?

Reply:
we usually use this construct which is Mac compatible I think:

```
TEMPDIR=$(mktemp -d)
function finalise {
    CODE=$?
    rm -rf "$TEMPDIR"
    exit $CODE
}
trap finalise EXIT
```

Reply:
`$HOME` aslo doesn't work on macos :-(

Reply:
I used this idiom for curl auth headers. it is like our current pattern with a tiny tweak so the header can be provided directly or indirectily via file

```
# The authentication header for curl commands, which *may* be a file ref if
# prefixed with@
BEARER_TOKEN_HEADER=@~/.datatrails/bearer-token.txt
```

But the key detail is that the OS sensitive bits need to be configurable and not baked into the examples themselves

Reply:
I've added a note regarding changing the config to refer to the OS temp location